aws/scripts/sh/batch_lhci_targets.sh

```
@@ -0,0 +1,226 @@
+#!/bin/bash
+
+# Lighthouse and Memory Leak Tests Batch Script
+# Adds lighthouse-desktop, lighthouse-mobile, and test-memory-leak targets to Makefile
+
+set -e
+
+echo "#### Adding lighthouse and memory leak test targets to Makefile"
+
+# Add Memory Leak Test Target
+cat >> Makefile << 'MEMORY_LEAK_TARGET'
```

Shellcheck (0.10.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)